Cargo.toml

```diff
@@ -22,3 +22,8 @@ default-members = [
 ]
 exclude = ["admin/rustfmt"]
 resolver = "2"
+
+[profile.release-lto]
```

Comment:
Should we use the `bench` profile for this?

Reply:
Good idea!